content/post/Android_Root101.en.md

```diff
@@ -1,31 +1,32 @@
 +++
 title = "A Guide to Flashing Custom ROMs [Android]"
-slug = "Flashing Custom ROMs 101"
+slug = "csrom101"
 date = "2018-02-25T00:00:00"
```

Comment:
2018??

Reply:
@rohanvtk It seems like you missed this bit.